datasource/netcdf.py

```diff
@@ -158,35 +158,33 @@ def get_available_parameters(object_type):
 PUMPLIKE_OBJECTS = ['pumpstation', 'pumpline']
 
 
-def get_variables(object_type=None, parameters=[]):
+# TODO: use this for pumps still? Currently unused.
+def get_variable(object_type=None, parameter=None):

```

Comment:
Throughout the code we use variable and parameter, better select one (variable?) and use this consequently